README.md

```diff
@@ -115,7 +115,7 @@ You would need 2 terminals for this setup
       
 2. **Run the insights proxy based on Linux or Mac**
 ```
-   SPANDX_CONFIG=/path/to/service_portal_api/dev_configs/service-portal_mac.js bash /path/to/insights-proxy/scripts/run.sh
+   SPANDX_CONFIG=/path/to/service_portal_api/config/spandx/service-portal_mac.js bash /path/to/insights-proxy/scripts/run.sh
```

Comment:
@mkanoor This looks correct, but on the diff for moving the files it seems they were moved to `config/spandx/dev_configs/<js file>.js` which doesn't match the docs above: `config/spandx/<js file>.js`

Reply:
Right, the files should just live in `config/spandx`.